apps/test/unit/applab/getListDropdownTest.js

```diff
@@ -0,0 +1,24 @@
+import {expect} from '../../util/configuredChai';
+
+var getListDropdown = require('@cdo/apps/applab/getListDropdown');
+
+describe('getListDropdown', function() {
```

Comment:
I wonder if there is a way to use test-specific data, so that this test would still pass if we change the data without changing any implementation (like how we test against all-the-things levels), but definitely not ship-blocking.